packages/babel-plugin-transform-runtime/test/fixtures/relativeRuntime/relative/output.js

```diff
@@ -0,0 +1,7 @@
+var _classCallCheck = require("../../../../node_modules/@babel/runtime/helpers/classCallCheck");
```

Comment:
I think that this is wrong. Even if it is resolved starting from `./subfolder`, the location of `@babel/runtime` relative to this file has only three `../../../`.

I would expect a test for the string version of this option to have another `node_modules` folder inside `./subfolder`: then this path should look like
```js
var _classCallCheck = require("./subfolder/node_modules/@babel/runtime/helpers/classCallCheck");
```